Profiles - ZIB 2017/zib-LaboratoryTestResult-Observation.xml

```diff
@@ -47,6 +48,7 @@
       <path value="Observation" />
       <short value="LaboratoryTestResult" />
       <definition value="Measurements and simple assertions made about a patient, device or other subject. These may be a simple or a complex observations. Simple Observations could consist of just a code and a value, whereas complex observations like panels and batteries could have related Observations and or components.&#xD;&#xA;The HCIM LaboratoryResult also supports simple and more complex situations. For a simple result there will be a single LaboratoryTestResult.LaboratoryTest.TestName in Observation.code and no LaboratoryTestresult.Test. For a more complex situation there will be a LaboratoryTestResult.Test and any number of related LaboratoryTestResult.LaboratoryTest, e.g. parts of a panel or battery. Each related LaboratoryTest shall be conveyed in Observation.related or Observation.component. The deciding factor in choosing between related or component is: does the observation have use as independent observation (Observation.related) or is the observation only relevant in the context of its parent (Observation.component).&#xD;&#xA;Note that (textual) reports on individual (panels of) observations, like conclusions, go into a different resource DiagnosticReport. The DiagnosticReport then contains the references to the Observation resources it concerns." />
+      <comment value="This profile is used both for the LaboratoryTestResult concept and the RelatedResult concept of HCIM LaboratoryTestResult. A LaboratoryTestResult can contain mutiple related RelatedResults. This is implemented using a parent Observation with related child Observations. See the code and related elements." />

```

Comment:
This text is incorrect, it talks about RelatedResults instead of LaboratoryTests. It should align with Observation.related, but perhaps be more consise. How about:
"This profile is used both for the root concept of HCIM LaboratoryTestResult and fir the LaboratoryTest concept of this HCIM. See Observation.code and Observation.related on guidance how to implement this."